packages/nodes-base/nodes/ExecuteWorkflow/constants.ts

```diff
@@ -0,0 +1,36 @@
+import type { FieldType } from 'n8n-workflow';
+
+export const INPUT_SOURCE = 'inputSource';
+export const WORKFLOW_INPUTS = 'workflowInputs';
+export const INPUT_OPTIONS = 'inputOptions';
+export const VALUES = 'values';
+export const JSON_EXAMPLE = 'jsonExample';
+export const TYPE_OPTIONS: Array<{ name: string; value: FieldType | 'any' }> = [
+	{
+		name: 'Allow Any Type',
+		value: 'any',
+	},
+	{
+		name: 'String',
+		value: 'string',
+	},
+	{
+		name: 'Number',
+		value: 'number',
+	},
+	{
+		name: 'Boolean',
+		value: 'boolean',
+	},
+	{
+		name: 'Array',
+		value: 'array',
+	},
+	{
+		name: 'Object',
+		value: 'object',
+	},
+	// Intentional omission of `dateTime`, `time`, `string-alphanumeric`, `form-fields`, `jwt` and `url`
+];
+
+export const DEFAULT_PLACEHOLDER = null;
```

Comment:
This is me, but we'll want to change this to `DEFAULT_FALLBACK_VALUE` or something